src/main/resources/stsjorbsmodResources/localization/eng/JorbsMod-Card-Strings.json

```diff
@@ -197,7 +197,7 @@
   },
   "stsjorbsmod:DemonicCoup": {
     "NAME": "Demonic Coup",
-    "DESCRIPTION": "stsjorbsmod:Ephemeral. NL Gain !M! Ritual. NL stsjorbsmod:Exert."
+    "DESCRIPTION": "Retain. NL When this card is Retained, gain !stsjorbsmod:MetaMagic! Ritual. NL Gain !M! Ritual. NL stsjorbsmod:Exert."
```

Comment:
Now that I've compared against Windmill Strike, consider swapping `When this card is Retained, gain !stsjorbsmod:MetaMagic! Ritual.` and `Gain !M! Ritual.` to be consistent.

Reply:
Like it is now?

Reply:
Yeah. Though, now the Spanish should change to match.